.gitignore

```diff
@@ -57,3 +57,8 @@ configure.sh
 /unittests/literal_true_false/literal_true_false
 /unittests/msexchange_noservice_wsdl/msexchange_noservice_wsdl
 
+*.o
+*.pro.user
+*.so
+*.so.*
+moc_*
```

Comment:
I think these should be at the top with the other build artifacts. It is a bit nit-picky, but I think it makes sense ;-)

Reply:
Makes sense, fixed, and added *.exe suffix.